depthai_sdk/examples/imu.py

```diff
@@ -0,0 +1,15 @@
+from depthai_sdk.classes.packets import IMUPacket
```

Comment:
Could IMUPacket be accessible from depthai_sdk? Not sure pros/cons of it

Reply:
Done